sections/collage.liquid

```diff
@@ -313,18 +313,20 @@
                 </div>
               </modal-opener>
 
-              <modal-dialog id="PopupModal-{{ block.id }}" class="collage-video__modal">
+              <modal-dialog id="PopupModal-{{ block.id }}" class="collage-video__modal media-modal">
                 <div class="collage-video__modal-content" role="dialog" aria-label="{{ block.settings.description | escape }}" aria-modal="true" tabindex="-1">
                   <button id="ModalClose-{{ block.id }}" type="button" class="collage-video__modal-toggle" aria-label="{{ 'accessibility.close' | t }}">{% render 'icon-close' %}</button>
-                  <deferred-media class="collage-video__modal-video template-popup">
-                    <template>
-                      {%- if block.settings.video_url.type == 'youtube' -%}
-                        <iframe src="https://www.youtube.com/embed/{{ block.settings.video_url.id }}?enablejsapi=1" class="js-youtube" allow="autoplay; encrypted-media" allowfullscreen title="{{ block.settings.description | escape }}"></iframe>
-                      {%- else -%}
-                        <iframe src="https://player.vimeo.com/video/{{ block.settings.video_url.id }}" class="js-vimeo" allow="autoplay; encrypted-media" allowfullscreen title="{{ block.settings.description | escape }}"></iframe>
-                      {%- endif -%}
-                    </template>
-                  </deferred-media>
+                  <div class="collage-video__modal-content-info">
```

Comment:
I had to move the padding-top that was on the deferred media element somewhere else so it wouldn't get excluded from the dismiss logic. The other 2 modaldialogs have some form of this content wrapper here so I figured I'd match that structure.